tembo-operator/src/stacks/templates/api.yaml

```diff
@@ -1,7 +1,10 @@
 name: API
 description: Tembo Stack with REST and graphQL interfaces.
-repository: "quay.io/tembo"
```

Comment:
You might want to add back the repository section. Even though there is a default, I think leaving this here is necessary.